onnxruntime/core/graph/graph_flatbuffers_utils.cc

```diff
@@ -0,0 +1,406 @@
+  type_proto.Clear();
+  FBS_SET_STR_VAL(type_proto, set_denotation, fbs_type_info.denotation());
+  auto value_type = fbs_type_info.value_type();
+  if (value_type == fbs::TypeInfoValue_tensor_type) {
+    auto fbs_tensor_type = fbs_type_info.value_as_tensor_type();
+    ORT_RETURN_IF_NOT(nullptr != fbs_tensor_type, "fbs_tensor_type cannot be null");
+    ORT_RETURN_IF_ERROR(LoadTensorTypeAndShapeOrtFormat(*fbs_tensor_type, *type_proto.mutable_tensor_type()));
+  } else {
+    return ORT_MAKE_STATUS(ONNXRUNTIME, INVALID_ARGUMENT, "Type:", value_type, " is not supported for now");
+  }
+
+  return Status::OK();
+}
+
+Status LoadValueInfoOrtFormat(const fbs::ValueInfo& fbs_value_info,
+                              ONNX_NAMESPACE::ValueInfoProto& value_info_proto) {
+  value_info_proto.Clear();
+
+  FBS_SET_STR_VAL(value_info_proto, set_name, fbs_value_info.name());
+  FBS_SET_STR_VAL(value_info_proto, set_doc_string, fbs_value_info.doc_string());
+
+  auto fbs_type_info = fbs_value_info.type();
+  ORT_RETURN_IF_NOT(nullptr != fbs_type_info, "fbs_type_info cannot be null");
+  ORT_RETURN_IF_ERROR(LoadTypeInfoOrtFormat(*fbs_type_info, *value_info_proto.mutable_type()));
+
+  return Status::OK();
+}
+
+onnxruntime::common::Status LoadAttributeOrtFormat(const fbs::Attribute& fbs_attr,
+                                                   ONNX_NAMESPACE::AttributeProto& attr_proto,
+                                                   std::unique_ptr<onnxruntime::Graph>& sub_graph,
+                                                   Graph& graph, Node& node,
+                                                   const logging::Logger& logger) {
+  attr_proto.Clear();
+  FBS_SET_STR_VAL(attr_proto, set_name, fbs_attr.name());
+  FBS_SET_STR_VAL(attr_proto, set_doc_string, fbs_attr.doc_string());
+  auto type = static_cast<AttributeProto_AttributeType>(fbs_attr.type());
+  attr_proto.set_type(type);
+  switch (type) {
+    case AttributeProto_AttributeType_FLOAT: {
+      attr_proto.set_f(fbs_attr.f());
+    } break;
+    case AttributeProto_AttributeType_INT: {
+      attr_proto.set_i(fbs_attr.i());
+    } break;
+    case AttributeProto_AttributeType_STRING: {
+      auto fbs_str = fbs_attr.s();
+      ORT_RETURN_IF_NOT(nullptr != fbs_str, "fbs_str cannot be null");
+      attr_proto.set_s(fbs_str->str());
+    } break;
+    case AttributeProto_AttributeType_TENSOR: {
+      auto fbs_tensor = fbs_attr.t();
+      ORT_RETURN_IF_NOT(nullptr != fbs_tensor, "fbs_tensor cannot be null");
+      ORT_RETURN_IF_ERROR(LoadInitializerOrtFormat(*fbs_tensor, *attr_proto.mutable_t()));
+    } break;
+    case AttributeProto_AttributeType_GRAPH: {
+      auto fbs_graph = fbs_attr.g();
+      ORT_RETURN_IF_NOT(nullptr != fbs_graph, "fbs_graph cannot be null");
+      attr_proto.mutable_g()->set_name("Empty graph proto from deserialization of ORT format model");
+      ORT_RETURN_IF_ERROR(Graph::LoadFromOrtFormat(*fbs_graph, graph, node, logger, sub_graph));
```

Comment:
Maybe add some info saying this is deliberate as we serialize the Graph instance for the subgraph separately, and don't use/need a GraphProto, but we need the attribute to exist in a full build so the ONNX checker is happy